charts/karpenter/README.md

```diff
@@ -15,7 +15,8 @@ $ helm upgrade --install karpenter karpenter/karpenter --namespace karpenter \
   --create-namespace --set serviceAccount.create=false --version 0.5.1 \
   --set controller.clusterName=${CLUSTER_NAME} \
   --set controller.clusterEndpoint=$(aws eks describe-cluster --name ${CLUSTER_NAME} --query "cluster.endpoint" --output json) \
-  --wait # for the defaulting webhook to install before creating a Provisioner 
+  --set controller.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME} \
```

Comment:
FYI, this is generated by `make helm`. You might already know this, since it looks correctly autogenerated.